### ISSUE TITLE
train

### ISSUE THREAD

**Issue by ZZFanya-DWR** [TencentARC/GFPGAN#19]
This training process looks a bit strange, is there something wrong with my configuration file, I am based on 1024 resolution

> 021-07-13 16:20:56,380 INFO: [train..][epoch:  0, iter:     100, lr:(2.000e-03,)] [eta: 112 days, 17:17:33, time (data): 1.796 (0.015)] l_g_pix: inf l_p_8: nan l_p_16: nan l_p_32: nan l_p_64: nan l_p_128: nan l_p_256: nan l_p_512: inf l_p_1024: inf l_g_percep: inf l_g_style: inf l_g_gan: nan l_g_gan_left_eye: nan l_g_gan_right_eye: nan l_g_gan_mouth: nan l_g_comp_style_loss: nan l_identity: nan l_d: nan real_score: nan fake_score: nan l_d_left_eye: nan l_d_right_eye: nan l_d_mouth: nan

**Comment by xinntao** [comment]
try to debug it, it seems that the loss is too large, do you use a pretrained model ~

**Comment by BruceZhou95** [comment]
> This training process looks a bit strange, is there something wrong with my configuration file, I am based on 1024 resolution
> 
> > 021-07-13 16:20:56,380 INFO: [train..][epoch:  0, iter:     100, lr:(2.000e-03,)] [eta: 112 days, 17:17:33, time (data): 1.796 (0.015)] l_g_pix: inf l_p_8: nan l_p_16: nan l_p_32: nan l_p_64: nan l_p_128: nan l_p_256: nan l_p_512: inf l_p_1024: inf l_g_percep: inf l_g_style: inf l_g_gan: nan l_g_gan_left_eye: nan l_g_gan_right_eye: nan l_g_gan_mouth: nan l_g_comp_style_loss: nan l_identity: nan l_d: nan real_score: nan fake_score: nan l_d_left_eye: nan l_d_right_eye: nan l_d_mouth: nan

Hello, may I ask where needs to be modified when using 1024*1024  images for training, looking forward to your reply, thanks
